test/unit/user/user.spec.js

```diff
@@ -9,8 +9,25 @@ import {
   successfulSignup,
   successfulLogin,
   nonExistingEmail,
-  incorrectPassword
+  incorrectPassword,resetPasswordSuccessful,
```

Comment:
Each of these variables should be on a separate line, or what do you think?

Reply:
That's true.